ledger-tool/src/main.rs

```diff
@@ -1156,8 +1120,6 @@ fn main() {
                 .arg(&hard_forks_arg)
                 .arg(&max_genesis_archive_unpacked_size_arg)
                 .arg(&snapshot_version_arg)
-                .arg(&maximum_full_snapshot_archives_to_retain)
-                .arg(&maximum_incremental_snapshot_archives_to_retain)
```

Comment:
I have no issue with removing the args - the flags were only added to avoid the exact scenario you described where someone creates a snapshot with `ledger-tool`, but the creation of new snap removes an old one unintentionally.

Adding it as a flag did give someone the knob to have ledger-tool auto delete oldest, but I've personally never used the flag and I have never seen use of that flag in restart instructions, guides, etc